### PR TITLE
Subscribing to notifications documentation

### PR DESCRIPTION
**Pull request by nikolajlauridsen**
This migrates the ["Subscribing to events"](https://our.umbraco.com/documentation/Fundamentals/Code/Subscribing-To-Events/) document over to the new notification pattern for V9. 

I've created a separate page for this, just like it was done with the [Reference/Notifications](https://our.umbraco.com/documentation/Reference/Notifications/) page, however, I don't know if it's confusing in this context or? Might be worth adding a note at the top of both documents that this is only for V9? 

I tried updating the links as well, however the "Components & Composing" page doesn't exist for V9 yet, so that will have to be edited at a later point.

### CODE REVIEW

**Comment by jonat123** [comment]
@nikolajlauridsen This looks good :) 

I agree that a small note on top of this article explaining that events have been renamed to notifications in V9 and then a link to the V8 article would make sense :) 

Regarding the "Components & Composing" page that this article links to, I think we should either wait to merge this article until it has been written or remove it until that page has been created so we don't end up having broken links in the documentation.

Let me know what you think 🙂

**Comment by nikolajlauridsen** [comment]
> Regarding the "Components & Composing" page that this article links to, I think we should either wait to merge this article until it has been written or remove it until that page has been created so we don't end up having broken links in the documentation.
> 
> Let me know what you think 🙂

Hmm giving it a bit more thought I think it makes sense to just remove it completely, to be honest since this article no longer references neither components nor composing. It's generally preferred to use the Startup file in Netcore.

It's worth mentioning though that it is still entirely possible to register your notification handlers with a composer, however, it's only really recommended for packages, so I chose not to mention it here since packages seems a bit out of scope for this article, [it's mentioned in the notifications reference docs though](https://our.umbraco.com/documentation/Reference/Notifications/#registering-notification-handlers-in-a-composer). 

TL;DR, I'll remove the link 🙃

**Comment by nikolajlauridsen** [comment]
I've removed the link now and added a note to the two notifications index docs.  👍

File: Fundamentals/Code/Subscribing-To-Notifications/index.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+meta.description: "Subscribing to notifications allows you to execute custom code on a number of operations both before and after the operation occurs"
+versionFrom: 9.0.0
+updated-links: true
+verified-against: beta-3
+state: complete
+v8-equivalent: "https://our.umbraco.com/documentation/Fundamentals/Code/Subscribing-To-Events/"
+---
+
+:::note
+Events was renamed to Notifications in V9, this article is only relevant for V9. For V8 equivalent see: [Events](../Subscribing-To-Events)
```
Comment (jonat123) [inline Fundamentals/Code/Subscribing-To-Notifications/index.md:12]:
```suggestion
In Umbraco 9 Events has been renamed to Notifications. This article is only relevant if you are using Umbraco 9. 
For Umbraco 8 see the [Subscribing to events article](../Subscribing-To-Events)
```

File: Reference/Notifications/index.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ versionFrom: 9.0.0
 verified-against: beta-2
 ---
 
+:::note
+Events was renamed to Notifications in V9, this article is only relevant for V9. For V8 equivalent see: [Events](../Events)
```
Comment (jonat123) [inline Reference/Notifications/index.md:8]:
```suggestion
In Umbraco 9 Events has been renamed to Notifications. This article is only relevant if you are using Umbraco 9.
For V8 equivalent see: [Events](../Events)
```

**Comment by jonat123** [comment]
@nikolajlauridsen I have made some small adjustments to the two notes that you made, let me know what you think 🙂

**Comment by nikolajlauridsen** [comment]
> @nikolajlauridsen I have made some small adjustments to the two notes that you made, let me know what you think 🙂

Looks good to me, nicer than what I came up with 😅

**Comment by jonat123** [comment]
Perfect 👌

All looks good to me, so lets get this merged in :D
